mne/io/brainvision/brainvision.py

```diff
@@ -358,10 +348,10 @@ def _get_vhdr_info(vhdr_fname, eog, misc, response_trig_shift, scale):
             highpass.append(line[5])
             lowpass.append(line[6])
         if len(highpass) == 0:
-            info['highpass'] = None
+            pass
         elif all(highpass):
             if highpass[0] == 'NaN':
-                info['highpass'] = None
+                pass
```

Comment:
I left these for readability and placeholders if we decide to handle them different in the future.